src/components/AccountIcon.js

```diff
@@ -14,55 +14,52 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-'use strict'
+'use strict';
 
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import { Image, View, StyleSheet } from 'react-native'
-import { blockiesIcon } from '../util/native'
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Image, View, StyleSheet } from 'react-native';
+import { blockiesIcon } from '../util/native';
 
-async function displayIcon (self, seed) {
+async function displayIcon(self, seed) {
   try {
-    let icon = await blockiesIcon(seed)
+    let icon = await blockiesIcon(seed);
     self.setState({
```

Comment:
You could put the `displayIcon` function inside AccountIcon class, and just use `this.setState`